lnwallet/size.go

```diff
@@ -0,0 +1,117 @@
+package lnwallet
```

Comment:
I don't think _any_ of the functions within this package need to be publicly exported. I only foresee this function being used within the state machine itself.